### ISSUE TITLE
Fully support less-loader ^12

### ISSUE THREAD

**Issue by D3strukt0r** [symfony/webpack-encore#1265]
When using webpack with less loader ^12 I get the following warning:

```
 WARNING  Webpack Encore requires version ^11.0.0 of less-loader. Your version 12.2.0 is too new. The related feature *may* still work properly. If you have issues, try downgrading the library, or upgrading Encore.
```

the only breaking change, as per the [release page](https://github.com/webpack-contrib/less-loader/releases/tag/v12.0.0) is: `minimum supported Node.js version is 18.12.0`

So i don't see the need for this warning. Could this be removed, or is there a reason for this?

**Comment by karpilin** [comment]
this is fixed in a long standing PR #1258

**Comment by Kocal** [comment]
Will be released with 4.17
